### PR TITLE
fixed 404s in /master/contrib/dockerfile and /master/docs/tutorial/do…

### PR DESCRIPTION
**Pull request by Science21330**
as far as i understand what's going on, this should work now

### CODE REVIEW

**Comment by tardyp** [comment]
please squash all those commits, so that the history is clean
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by tardyp** [comment]
:+1:

File: master/docs/tutorial/docker.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Building and running Buildbot
 .. code-block:: bash
 
   # Download Buildbot Dockerfile.
-  wget https://raw.github.com/buildbot/buildbot/master/master/contrib/Dockerfile
+  wget raw.githubusercontent.com/buildbot/buildbot/eight/master/contrib/Dockerfile
```
Comment (pquentin) [inline master/docs/tutorial/docker.rst]: There's an extra unneeded space here.

**Comment by pquentin** [comment]
Sounds good, not sure about the branch to choose here. (Not using buildbot anymore, but @mention-bot mentioned me.)

File: master/docs/tutorial/docker.rst (hunk 1 above)
Comment (Science21330) [inline master/docs/tutorial/docker.rst]: *was

**Comment by Science21330** [comment]
@pquentin the origin was eight, my fork is fixed404s (not sure on the specific info you need, sort of new to github, sorry)

**Comment by tardyp** [comment]
Eight is looking for a new maintainer. Please step in if you need more eight support. closing for now

**Pull request closed without merging**